db/seeds.rb

```diff
@@ -12,103 +12,120 @@
                              status: "active",
-                             categories: [entree_cat])
+                             categories: [entree_cat],
+                             image: File.open('public/UnicornOnTheCob.jpg'))
 
 medusa         = Item.create(title: "Medusa Hair Pasta",
                              description: "Freshly prepared by Perseus, our world-renowned and legendary (be)head chef",
                              price: 12.99,
                              status: "active",
-                             categories: [pasta_cat, entree_cat])
+                             categories: [pasta_cat, entree_cat],
+                             image: File.open('public/MedusaHairPasta.jpg'))
 
 pixie          = Item.create(title: "Pixie Bread Stix",
                              description: "Who knew bread sticks could make you feel so spritely!",
                              price: 4.75,
                              status: "active",
-                             categories: [appetizer_cat])
+                             categories: [appetizer_cat],
+                             image: File.open('public/PixieBreadStix.jpeg'))
 
 phoenix        = Item.create(title: "Phoenix Spicy Curry",
                              description: "One bite and you'll feel reborn",
                              price: 11.50,
                              status: "active",
-                             categories: [curry_cat, entree_cat])
+                             categories: [curry_cat, entree_cat],
+                             image: File.open('public/PhoenixSpicyCurry.jpg'))
 
 titan          = Item.create(title: "Titan Burger",
                              description: "So big and powerful it'll descend upon your tastebuds in an epic clash",
                              price: 9.25,
                              status: "active",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/TitanBurger.jpg'))
 
 mermaid        = Item.create(title: "Filet of Mermaid",
                              description: "Only wild-caught...no farming here!",
                              price: 13.00,
                              status: "retired",
-                             categories: [seafood_cat, entree_cat])
+                             categories: [seafood_cat, entree_cat],
+                             image: File.open('public/FiletOfMermaid.jpg'))
 
 minotaur       = Item.create(title: "NY Strip Minotaur",
                              description: "A dish served as rare as the beast itself",
                              price: 19.99,
                              status: "active",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/NYStripMinotaur.jpg'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.